options.go

```diff
@@ -103,6 +107,71 @@ func (opt *Options) init() {
 	}
 }
 
+func ParseURL(sURL string) (*Options, error) {
+
+	parsedUrl, err := url.Parse(sURL)
+	if err != nil {
+		return nil, err
+	}
+
+	// scheme
+	if parsedUrl.Scheme != "postgres" {
+		return nil, errors.New("Invalid scheme: " + parsedUrl.Scheme)
```

Comment:
Can we add `pg: ` prefix to these errors? E.g. `fmt.Errorf("pg: invalid scheme: %s", parsedUrl.Scheme)`